scripts/src/internal/routes.ts

```diff
@@ -1,20 +1,20 @@
-import { Request, Response, Router, Express } from "express";
+import { Request, Response, Router, Express, RequestHandler } from "express";
 
 import {
 	CompletedPayment,
 	paymentComplete as paymentCompleteService,
 	paymentFailed as paymentFailedService,
 } from "./services";
 
-export async function paymentComplete(req: Request, res) {
+export const paymentComplete = async function(req: Request, res: Response) {
 	await paymentCompleteService(req.body as CompletedPayment, req.logger);
 	res.status(200).send({ status: "ok" });
-}
+} as any as RequestHandler;
```

Comment:
this is ugly but needed.
the problem is that the compiler has issues with it because we're not specifying the last (`next`) param.

Reply:
so lets define the next parameter and just not use it. It's better than this

Reply:
no, it's not better.  this "hack" is ts only, but the js code will be clean.
adding the `next` param will cause it to also be there in the js code.